src/templates/article.js

```diff
@@ -177,10 +178,16 @@ const Article = props => {
         <Layout>
             <Helmet>
                 <title>{articleTitle}</title>
-                <meta property="og:title" content={articleTitle} />
-                <meta property="og:url" content={articleUrl} />
-                <meta property="og:type" content="article" />
-                <link rel="canonical" href={articleUrl} />
+                <meta
+                    property="og:image"
+                    content={
+                        og.image || `${siteUrl}${withPrefix(meta['atf-image'])}`
```

Comment:
I don't think we should fallback to the atf - they want to be specific with the image used for sharing. We *do* currently have a fallback in the `TwitterMeta` component to match the `og:` image with the `twitter:` image which should be sufficient if the `og:image` isn't specified. We'll need to ensure that the twitter component doesn't over-ride the `og:image` though